scripts/auth-test-harness.js

```
@@ -0,0 +1,463 @@
+
+      if (!siteResponse.data) {
+        console.error('❌ Failed to create test site');
+        return false;
+      }
+
+      console.log(`✅ Created test site: ${siteResponse.data.name}`);
+      var siteSlug = siteResponse.data.slug;
+    } else {
+      console.log(`Found ${sitesResponse.data.length} sites`);
+      var siteSlug = sitesResponse.data[0].slug;
+    }
```

Biome (1.9.4)

[error] Shouldn't redeclare 'siteSlug'. Consider to delete it or rename it.

'siteSlug' is defined here:


(lint/suspicious/noRedeclare)